backend/app/main.py

```
@@ -2,13 +2,15 @@
 import time
 from contextlib import asynccontextmanager
 from pathlib import Path
+from typing import AsyncGenerator
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `AsyncGenerator`

Import from `collections.abc`

(UP035)